index.html

```diff
@@ -9,5 +9,25 @@
   <link rel="stylesheet"  type="text/css" href="application.css">
 </head>
 <body>
+
+    <div class="title-container">
+        <img width="50px" height="50px" src="./assets/triangle.svg" alt="triangle">
+        <p>Front-End Tools</p>
+    </div> 
+
+    <div class="tech-border">
+        <div class="tech-container">
+            <img height="100px" width="100px" src="./assets/react_icon.svg" alt="react">
+            <p>ReactJS</p>
+            <p>Release: 2013</p>
```

Comment:
Here you can use `<span/>` because it is an inline element. You should use `<p/>`when you have one paragraph.